buildkitd/Earthfile

```diff
@@ -12,7 +12,7 @@ buildkitd:
             ARG BUILDKIT_BASE_IMAGE=$BUILDKIT_PROJECT+build
         END
     ELSE
-        ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:8df7c52e603ee6db6a471fcb7f6a85a407f611f4+build
+        ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:40cd44f81962f99200f10df3a79d044f9df0aefe+build
```

Comment:
don't forget to change this to the rebased commit :)

Reply:
this still needs updating, earthly-main under buildkit is now at `af796f0cb8ff6cf53fdebd543ff1e7cd173c00a1`

Reply:
Thanks. Updated to `af796f0cb8ff6cf53fdebd543ff1e7cd173c00a1`